src/contribute/intellij.md

```diff
@@ -15,6 +15,22 @@ section_menu: section-menu/contribute.html
 >
 > Mastering IntelliJ is, ultimately, your responsibility.
 
+## Create a working Gradle module
+
+1. Create an empty IntelliJ project outside of the Beam source tree.
+2. Under Project Structure > Project, select a Project SDK.
+3. Under Project Structure > Modules, click the + sign to add a module and select "Import Module".
+    1. Select the directory containing the Beam source tree.
+    2. Tick the "Import module from external model" button and select Gradle from the list.
+    3. Tick the following boxes.
+       * Use auto-import
+       * Create separate module per source set
+       * Store generated project files externally
+       * Use default gradle wrapper
+
+This should result in a working Gradle project. Build the project by executing the "build" task in
```

Comment:
I believe we should tell users to delegate build/test actions to gradle. We should tell users to do this as well, instructions are here: https://dzone.com/articles/using-idea-to-delegate-build-and-run-actions-to-gr